blocks/init/src/Blocks/components/accordion/manifest.json

```diff
@@ -1,4 +1,5 @@
 {
+	"$schema": "./../../../../node_modules/@eightshift/frontend-libs/schemas/component.json",
```

Comment:
This will work:

```
https://raw.githubusercontent.com/infinum/eightshift-frontend-libs/develop/schemas/component.json
```

Just the GH one doesn't work, because that's the link to the website with the file, while this is a link to the raw file. 
I think we should add schemas to the `main` branch first so that you can link from that branch, since `main` is stable, and `develop` is not.

Reply:
I have updated this